java/test/src/main/java/io/ray/test/WorkerJvmOptionsTest.java

```diff
@@ -21,9 +21,10 @@ public void testJvmOptions() {
     // that raylet can correctly handle dynamic options with whitespaces.
     ActorHandle<Echo> actor =
         Ray.actor(Echo::new)
-            .setJvmOptions(ImmutableList.of("-Dtest.suffix=suffix", "-Dtest.suffix1=suffix1"))
+            .setJvmOptions(
+                ImmutableList.of("-Dtest.suffix=suffix", "-Dtest.suffix1=suffix1 suffix1"))
             .remote();
     ObjectRef<String> obj = actor.task(Echo::getOptions).remote();
-    Assert.assertEquals(obj.get(), "suffix");
+    Assert.assertEquals(obj.get(3 * 1000), "suffix");
```

Comment:
How about adding a new case `testJvmOptionsWithWhitespace()` instead of change this case?
Otherwise, this case will be changed by other guys in the future, with ignoring the whitespace.

Reply:
sure, but seems that this case is used to test if the argument contains whitespace according to the comments:
// The whitespaces in following argument are intentionally added to test
// that raylet can correctly handle dynamic options with whitespaces.